src/panels/Scene/SceneGraphNode/SceneGraphNodeHeader.tsx

```diff
@@ -56,7 +57,8 @@ export function SceneGraphNodeHeader({ uri, label, onClick }: Props) {
     <ThreePartHeader
       title={onClick ? titleButton : name}
       leftSection={
-        hasRenderable && <PropertyOwnerVisibilityCheckbox uri={renderableUri} />
+        leftSection ??
+        (hasRenderable && <PropertyOwnerVisibilityCheckbox uri={renderableUri} />)
```

Comment:
Hm, this would replace the checkbox, if a left section is specified.  This should be made clearer, e.g. using a comment above line 18

Reply:
Another option would be to include both the checkbox in a `Group`, but either way, the intention of `leftSection` needs to be communicated here